docs/reference/fields.txt

```diff
@@ -976,6 +1005,14 @@ setter methods for fields of your custom type.
    venue = Venue.new(location: point) # This uses the Point#mongoize instance method.
    venue = Venue.new(location: [ 12, 24 ]) # This uses the Point.mongoize class method.
 
+.. note::
+
+  The ``mongoize`` method should raise an "InvalidValue" error on values that
+  are uncastable to your custom type. Mongoid will handle catching that error
```

Comment:
Suggest "Mongoid will handle that error and write `nil`"  since "catch" is an operation in Ruby that is different from "rescue".

Reply:
done